backend/src/controllers/AuthController.js

```diff
@@ -166,7 +166,7 @@ export default class AuthController {
 
   static verifyEmail = async (req, res, next) => {
     try {
-      const { token, id } = req.query;
+      const { token, id } = req.body;
```

Comment:
Please why are you destructing the request body instead of query?

Reply:
The frontend guy said he will be sending the token and id as a body to the email verification controller

Reply:
okay. noted.